kratos/geometries/nurbs_curve_geometry.h

```diff
@@ -17,10 +17,9 @@
 
 // Project includes
 #include "geometries/geometry.h"
-
 #include "geometries/nurbs_shape_function_utilities/nurbs_curve_shape_functions.h"
 #include "geometries/nurbs_shape_function_utilities/nurbs_interval.h"
-
+#include "geometries/nurbs_shape_function_utilities/projection_nurbs_geometry_utilities.h"
```

Comment:
those things should not be in geomery IMO, but rather in utilites
Please wait for other opinions

Reply:
I agree with @philbucher, 

If you want them to be in geometries, please use the internal folder and manespace.

Reply:
Shall we create a sub folder in the utilities, because there will be a bit more utilities like that one?